lighthouse-core/config/default-config.js

```diff
@@ -428,6 +448,9 @@ const defaultConfig = {
         {id: 'timing-budget', weight: 0, group: 'budgets'},
         {id: 'resource-summary', weight: 0, group: 'diagnostics'},
         {id: 'third-party-summary', weight: 0, group: 'diagnostics'},
+        {id: 'uses-http2', weight: 0, group: 'diagnostics'},
+        {id: 'uses-passive-event-listeners', weight: 0, group: 'diagnostics'},
+        {id: 'no-document-write', weight: 0, group: 'diagnostics'},
```

Comment:
we might want to retitle this audit to be 

`Avoids document.write` / `Avoid document.write` to be more consistent with the other diagnostics

Reply:
Given that the title a user sees already has that wording, is it worth renaming the id? Also, the current id matches the web.dev article link but I don't know if that matters at all.

Reply:
> Given that the title a user sees already has that wording

The success title uses `Avoids` but the failure title is `Uses` to match the best practices norms of descriptive failure and success titles.  The other performance audits tend to use imperative failure titles to frame what action the user needs to take.

Definitely not trying to suggest we change the audit id over this :)

Reply:
Ah, ok. So just updating the wording on the failure title then?

Reply:
yep 👍  :)